### PR TITLE
Add Rmd Servo and HBridge actuator examples. WIP

### PR DESCRIPTION
**Pull request by ethanrublee**

### CODE REVIEW

File: circuitpy/lib/farm_ng/utils/can.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def setup_can_default():
     """Sets up the default canio.CAN connection, for your board type Uses baud_rate of 250K (default bus speed for
     the Amiga)"""
     board_type = get_board_type()
-    baud_rate = 250_000
+    baud_rate = 1_000_000
```
Comment (ethanrublee) [inline circuitpy/lib/farm_ng/utils/can.py:16]: TODO this needs to be easily changed from client code.

File: circuitpy/lib/farm_ng/actuators/rmd_servo.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    format = "<4Bi"
+    cmd_byte = 0xA2
+
+    def __init__(self, rpm: int = 0):
+        self.rpm = rpm
+        
+
+    def encode(self):
+        """Returns the data contained by the class encoded as CAN message data."""
+        speed = self.rpm * 600 # 0.01 degrees per second - 360 deg/60 seconds * 100
```
Comment (Hackerman342) [inline circuitpy/lib/farm_ng/actuators/rmd_servo.py:23]: oof - good catch

File: circuitpy/lib/farm_ng/utils/packet.py [hunk 3]
```diff
@@ -365,61 +365,6 @@ def __str__(self):
         return f"node_state: {self.node_state} ticks_ms: {self.ticks_ms} serial_number: {self.serial_number}"
 
 
-class RmdSpeedCommand(Packet):
```
Comment (Hackerman342) [inline circuitpy/lib/farm_ng/utils/packet.py:368]: So this is the pattern we want, with packets in the file they're most relevant to? @ethanrublee 
I'm good with it, just want to be sure so I can be consistent going forward.

File: circuitpy/lib/farm_ng/utils/can.py (hunk 1 above)
Comment (Hackerman342) [inline circuitpy/lib/farm_ng/utils/can.py:16]: pass as arg, don't change default

```
def setup_can_default(baud_rate int: 250_000):
    """Sets up the default canio.CAN connection, for your board type Uses baud_rate of 250K (default bus speed for
    the Amiga)"""
    board_type = get_board_type()
    if board_type == BoardType.AMIGA_DISPV0:
        ...
```

Comment (Hackerman342) [reply circuitpy/lib/farm_ng/utils/can.py:16]: Oh just saw thread started above

Comment (Hackerman342) [reply circuitpy/lib/farm_ng/utils/can.py:16]: can.py:
```
def setup_can_default(baud_rate: int = 250_000):
    """Sets up the default canio.CAN connection, for your board type Uses baud_rate of 250K (default bus speed for
    the Amiga)"""
    board_type = get_board_type()
    if board_type == BoardType.AMIGA_DISPV0:
        ...
```
main_loop.py:
```
class MainLoop:
    """Main driver for all farm-ng apps run on microcontrollers."""

    def __init__(self, AppClass, has_display=True, baud_rate: int =  250_000) -> None:
        ...
        self.can = setup_can_default(baud_rate)
        ...
```

example/foo.py
```
...
def main():
    MainLoop(AppClass=RmdServoApp, has_display=False, baud_rate=1_000_000).loop()

main()
```

**Comment by Hackerman342** [comment]
Closing as stale

**Pull request closed without merging**